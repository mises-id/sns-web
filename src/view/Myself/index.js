import React, { useReducer, useEffect, useRef, useState } from 'react';
const Myself = (props)=>{
  return <div>我的</div>
}
export default Myself