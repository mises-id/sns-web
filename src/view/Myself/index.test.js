import React from 'react';
import renderer from 'react-test-renderer';
import Myself from './';
jest.mock('react-i18next', () => ({
  useTranslation: () => ({t: key => key})
}));
describe('Myself', () => {
  it('should render correctly', () => {
    const wrapper = renderer.create(<Myself></Myself>).toJSON()
    expect(wrapper).toMatchSnapshot();
  });
});