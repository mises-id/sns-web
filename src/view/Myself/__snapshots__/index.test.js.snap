// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Myself should render correctly 1`] = `
<div>
  我的
</div>
`;
